operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java

```diff
@@ -176,6 +184,20 @@ protected Parameters doInvoke(FHIROperationContext operationContext, Class<? ext
         int maxPageSize = Math.max(1, FHIRConfigHelper.getIntProperty("fhirServer/core/maxPageSize", FHIRConstants.FHIR_PAGE_SIZE_DEFAULT_MAX));
         List<Entry> allEntries = new ArrayList<>(maxPageSize);
         allEntries.add(patientEntry);
+        List<String> resourceIds = new ArrayList<String>();
+        // Look up which extra resources should be returned
+        List<String> extraResources = FHIRConfigHelper.getStringListProperty("fhirServer/operations/everything");
```

Comment:
as mentioned in other comment, I would prefer an extra level here.  something like: `fhirServer/operations/everything/includeTypes`

additionally, we usually put constants for these config property names into FHIRConfiguration

Reply:
Done